backend/composer/services/cs_ingestion/helpers/anatomical_entities_helper.py

```diff
@@ -69,19 +69,19 @@ def add_entity_to_instance(instance, entity_field, entity, update_anatomic_entit
 
 def get_or_create_complex_entity(region_uri, layer_uri, update_anatomical_entities=False):
     try:
-        layer = Layer.objects.get(ontology_uri=layer_uri)
+        layer = Layer.objects.get(ae_meta__ontology_uri=layer_uri)
     except Layer.DoesNotExist:
         layer = None
 
     try:
-        region = Region.objects.get(ontology_uri=region_uri)
+        region = Region.objects.get(ae_meta__ontology_uri=region_uri)
     except Region.DoesNotExist:
         region = None
 
     if update_anatomical_entities:
         if not layer:
             try:
-                layer_meta = AnatomicalEntityMeta.objects.get(ontology_uri=layer_uri)
+                layer_meta = AnatomicalEntityMeta.objects.get(ontology_uri=layer_uri)                    
                 layer, _ = convert_anatomical_entity_to_specific_type(layer_meta, Layer)
```

Comment:
suggestion (non-blocking): We can rename this method now. It no longer 'converts' it creates.

Reply:
thanks! @afonsobspinto 👍